docs/dev/adding_instruments.rst

```diff
@@ -590,10 +677,26 @@ The code below shows how this can be accomplished with dynamic properties.
 
 
 Now our voltage property has a dynamic validity range, either [-1, 1] or [0, 1].
+A side effect of this is that the property's docstring should be less specific, to avoid it containing dynamically changed information (like the admissible value range).
 In this example, the property name was :code:`voltage` and the parameter to adjust was :code:`values`, so we used :code:`self.voltage_values` to set our desired values.
 
-Family of instruments with similar features
-*******************************************
+.. _instruments_with_similar_features:
+
+Instruments with similar features
+=================================
+
+When instruments have a similar set of features, it makes sense to use inheritance to obtain most of the functionality from a parent instrument class, instead of copy-pasting code.
+
+.. note::
+    Don't forget to update the instrument's :code:`name` attribute accordingly, by either supplying an appropriate argument (if available) during the :code:`super().__init__()` call, or by setting it anew below that call.
+
+Sometimes one only needs to add additional properties and methods.
+Often, some of the already present properties/methods need to be completely replaced by defining them again in the derived class.
+Often, however, only some details need to be changed.
+This can be dealt with efficiently using dynamic properties.
```

Comment:
I'm not happy with the duplication of "often"
```suggestion
In some cases, one only needs to add additional properties and methods.
In other cases, some of the already present properties/methods need to be completely replaced by defining them again in the derived class.
Often, however, only some details need to be changed.
This can be dealt with efficiently using dynamic properties.
```